### PR TITLE
Comparison: Maha and facebook/Duckling

### LINKED ISSUE #19: Add a section in the documentation to compare the parsing capability of Maha and Duckling
### What problem are you trying to solve?

This provides a comparison between Maha parsing capabilities and [Duckling](https://github.com/facebook/duckling), a known parsing library by Facebook. The results will help provide statistics on Maha's performance and the robustness of its parsing results.

### Examples (if relevant)

_No response_

### Definition of Done

- The comparison should include:
  1. Parsing results.
  2. Time taken to get the results.
  3. Memory consumption.
  5. Ease of use and integration.
  4. Any additional useful metric.
- Long and short texts should be tested.
- Duckling should be installed and used locally for fair comparison.
- Only compare the dimensions implemented in Maha, which are `time`, `numeral`, `ordinal`, `duration`.
- The comparison should be clear, concise and organized.

### PR DESCRIPTION
**Pull request by MuhammadAlBarham**
**What does this pull request change**?

- resolves #19.

**Status (please check what you already did)**:

- [ ] added some tests for the functionality
- [x] updated the documentation
- [x] `tox` passes

### CODE REVIEW

File: docs/source/comparison.rst [hunk 1]
```diff
@@ -0,0 +1,451 @@
+comparison
```
Comment (TRoboto) [inline docs/source/comparison.rst:1]:
```suggestion
Duckling
```

**Comment by TRoboto** [comment]
Thank you for your efforts. I have made improvements to the structure and language. You can see the changes here:
https://maha--55.org.readthedocs.build/en/55/

Please update the Mean Time for all examples as well as the examples in the colab notebook. Also, please confirm that all results are correct.

**Review by TRoboto — APPROVED** [review]
I believe this PR is ready to be merged. Please double check the changes and confirm that everything looks good.

**Review by MuhammadAlBarham — COMMENTED** [review]
LGTM, after I double-checked the PR.

**Pull request merged**